src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -72,6 +73,8 @@ void setUp() {
         recipeRepository.deleteAll();
     }
 
+    static final Logger logger = Logger.getLogger(RecipeRepositoryTests.class.getName());
```

Comment:
nit: this change feels unrelated to 82a77f60f5b1da17ae8d101bb50a325cc65d2b5e commit

As I understand, all these changes are related to fixing the test, so I'd mention both changes in commit message